src/any_agent/config.py

```diff
@@ -51,13 +51,22 @@ class MCPSseParams(BaseModel):
 
 class TracingConfig(BaseModel):
     model_config = ConfigDict(extra="forbid")
-    output_dir: str | None = "traces"  # None for no json saved trace
+    output_dir: str = "traces"  # None for no json saved trace
+    enable_console: bool = True
+    enable_file: bool = True
```

Comment:
No strong opinion but I would prefer to not have these args and instead rely on `None` value.

I know this is more explicit but I think is simple enough API for someone to understand `None = disabled`.
Also, some corner cases are opened and not handled here (i.e. `enable_console=True` + all the console options set to `None / False`)

Reply:
Similar to you I don't have a strong opinion here. My thinking about adding explicit enable/disable booleans is that as we add more args that relate to each log setting, it might get confusing about how to turn it off.

For example, if right now a person wanted to disable console logging but continue with file logging, its not clear how they would disable the console. Needing to set all values to None might be a bit confusing. 

I'm thinking this will also make more sense when we incorporate some sort of file rotation functionality into the Tracer, mentioned in https://github.com/mozilla-ai/any-agent/issues/175.

Reply:
Either way, you make a good point about the corner case of enable=True with all console options set to None